src/ng1/directives/infiniteScroll/infiniteScroll.controller.js

```diff
@@ -89,8 +89,10 @@ export class InfiniteScrollController {
         // repeat until we have enough items
         this._ensureScrollableInterval = this.$interval.interval(() => {
 
+            let isElementHidden = this.$element.is(':hidden');
+
             // if we are currently loading or have loaded all pages then do nothing
-            if (this.loading || this.complete || document.hidden) {
+            if (this.loading || this.complete || document.hidden || isElementHidden) {
```

Comment:
Hi Victor, thanks very much for the contribution, I wonder would we inline the `is` call. It can be quite an expensive check so adding it at the end of the if statement would allow us to only call it if all the other checks are false, eg:

```typescript
if (this.loading || this.complete || document.hidden || this.$element.is(':hidden')) {
```